### PR TITLE
ADD Microsoft sqlserver 2017

### PR DESCRIPTION
**Pull request by jagjag**
ADD mssqlserver linux editon in docker 2017-CU5

### CODE REVIEW

File: templates/mssqlserver/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      required: true
+      label: "SA Password"
+      description: "require Strong Password"
+      default: <YourStrong!Passw0rd>
+    - variable: "mssql_port"
+      type: "string"
+      required: true
+      label: "mssql_port"
+      description: "connect port"
+      default: 1433
```
[the remark this answers is not in the document]
Comment (jagjag) [reply templates/mssqlserver/0/rancher-compose.yml]: true.

File: templates/mssqlserver/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      required: true
+      label: "# of mssql instance "
+      description: "# of mssql instance"
+      default: 3
+    - variable: "mssql_password"
+      type: "string"
+      required: true
+      label: "SA Password"
+      description: "require Strong Password"
+      default: <YourStrong!Passw0rd>
```
[the remark this answers is not in the document]
Comment (jagjag) [reply templates/mssqlserver/0/rancher-compose.yml]: ok

File: templates/mssqlserver/0/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  version: 2017-CU5-rancher1
+  description: sqlserver
+  maintainer: "JagJag <kyukou@gmail.com>"
+  questions: 
+    - variable: "mssqlscale"
+      type: "int"
+      required: true
+      label: "# of mssql instance "
+      description: "# of mssql instance"
+      default: 3
```
[the remark this answers is not in the document]
Comment (jagjag) [reply templates/mssqlserver/0/rancher-compose.yml]: I see.   cluster doesn't support yet.     I will remove scale for this editon .

File: templates/mssqlserver/0/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  mssql-linux:
+    image: microsoft/mssql-server-linux:2017-CU5
+    environment:
+      ACCEPT_EULA: Y
+      MSSQL_SA_PASSWORD: ${mssql_password}
```
[the remark this answers is not in the document]
Comment (jagjag) [reply templates/mssqlserver/0/docker-compose.yml]: OK , got it.

**Comment by jagjag** [comment]
Hi  @rawmind0 , thanks for your review
ALL issue fixed ,please check the code . thank you

**Comment by rawmind0** [comment]
@jagjag , thanks for the changes.

LGTM...

**Pull request merged**